int16 num
int16 channel
int16 speed
int16 minVal
int16 maxVal
uint8 startPercents